web/src/components/dashboard/SidebarCardsList.test.tsx

```diff
@@ -62,4 +81,56 @@ describe("<SidebarCardsList />", () => {
     fireEvent.click(screen.getByTestId("hidden-opportunity-header"));
     expect(mockAnalytics.event.for_you_card_unhide_button.click.unhide_cards).toHaveBeenCalledTimes(1);
   });
+
+  describe("empty state messaging", () => {
+    it("displays Complete Required Tasks Message when there are no primary actions or opportunities", () => {
+      renderComponent({});
+      expect(screen.getByTestId("complete-required-tasks-msg")).toBeInTheDocument();
+      expect(screen.queryByText(Config.dashboardDefaults.emptyOpportunitiesText)).not.toBeInTheDocument();
+    });
+
+    it("does not display Complete Required Tasks Message when the For You section has cards", () => {
+      const topCards = [
+        generateSidebarCardContent({
+          id: "some-fake-top-card",
+          section: "above-opportunities",
+        }),
+      ];
+
+      const bottomCards = [
+        generateSidebarCardContent({
+          id: "some-fake-bottom-card",
+          section: "below-opportunities",
+        }),
+      ];
+      renderComponent({ topCards, bottomCards });
+
+      expect(screen.queryByTestId("complete-required-tasks-msg")).not.toBeInTheDocument();
+    });
+
+    it("does not display Complete Required Tasks Message when there are top cards", () => {
+      const topCards = [
+        generateSidebarCardContent({
+          id: "some-fake-top-card",
+          section: "above-opportunities",
+        }),
+      ];
+      renderComponent({ topCards, displayFundings: true, displayCertifications: true });
```

Comment:
pebble - we are rendering the component with `displayFundings: true` and `displayCertifications: true` but that is not reflected in the test name. We should update the title, or maybe create a separate test.

Reply:
Another option is to update the initial value of the renderComponet so `displayFundings: true` and `displayCertifications: true` by default